src/core_modules/capture-ui/UserAvatarImage/UserAvatarImage.component.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+const sizes = {
+    extrasmall: {
+        height: 24,
+        width: 24,
+    },
+    small: {
+        height: 36,
+        width: 36,
+    },
+    medium: {
+        height: 48,
+        width: 48,
+    },
+    large: {
+        height: 72,
+        width: 72,
+    },
+    extralarge: {
+        height: 144,
+        width: 144,
+    },
+};
+
+const styles = {
+    img: {
+        borderRadius: '50%',
+        objectFit: 'cover',
+    },
+    ...sizes,
+};
+
+type Props = {
+    imageUrl: string,
+    dataTest: string,
+    classes: Object,
+    className: Object,
+    size: 'extrasmall' | 'small' | 'medium' | 'large' | 'extralarge',
+};
+
+const UserAvatarImagePlain = ({ imageUrl, dataTest, classes, className, size }: Props) => (
```

Comment:
@henrikmv,
I agree with the usage of a made UI component. In this use case, the displayed image is the [first TEA ](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/CardList/CardList.selectors.js#L13)of type `IMAGE` with a value, while the `UserAvarar` component in the UI library is intended for the user logged into the system.
The only suggestion I have is to change the name of the component to something more general, not related to the user or avatar. Can it be renamed to something like `CardImage` or just `Image`? WDYT?
Thank you!

Reply:
Thank you! I changed to CardImage